src/test/java/duke/UiTest.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import duke.ui.Ui;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+public class UiTest {
+    private static final String SEPARATOR = System.getProperty("line.separator");
+    private static final String LOGO = " _   _       _   _ _            \n" +
+            "| | | |     | | | (_)           \n" +
+            "| |_| | ___ | |_| |_ _ __   ___ \n" +
+            "|  _  |/ _ \\| __| | | '_ \\ / _ \\\n" +
+            "| | | | (_) | |_| | | | | |  __/\n" +
+            "\\_| |_/\\___/ \\__|_|_|_| |_|\\___| \n";
+    private static final String DIVIDER = "<<>><<>><<>><<>><<>><<>><<>><<>><<>><<>><<>><<>><<>><<>><<>>";
+    private static final String WELCOME_MESSAGE = DIVIDER + "\n Thanks for contacting Hotline! \n" +
+            " How can I help you today? \n"
+            + DIVIDER + "\n";
+    private static final String GOODBYE_MESSAGE = "██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████\n" +
+            "██                  ████      ████████  ██████████████████      ████    ██  ▒▒██████████  ████████  ████████        ░░██    ██████████\n" +

```

Comment:
Ryan Tan from 1231 says hi

Reply:
Thanks for mentioning. I say hi as well :smiley:, but it would probably be a good idea not to mention miscellaneous things inside review comments.